static/src/javascripts/projects/common/modules/experiments/tests/measure-understanding.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+
+import { addEventListener } from 'lib/events';
+import config from 'lib/config';
+import fastdom from 'lib/fastdom-promise';
+import mediator from 'lib/mediator';
+import { local as localStorage } from 'lib/storage';
+import template from 'lodash/utilities/template';
+import ophan from 'ophan/ng';
+import measureUnderstandingStr
+    from 'raw-loader!common/views/experiments/measure-understanding.html';
+import { markup as thumb } from 'svg-loader!svgs/icon/thumb.svg';
+
+const MeasureUnderstanding = () => {
+    // Test id
+    const id = 'MeasureUnderstanding';
+
+    // Ophan survey id
+    // TODO get survey id
+    const component = 'data_survey_1234';
+
+    // will run in articles only if no story questions atom is already there
+    // *and* the user hasn't already answered the question
+    const canRun = () =>
+        config.page.contentType === 'Article' &&
+        !config.page.isAdvertisementFeature &&
+        !localStorage.get(id) &&
+        !document.getElementsByClassName('js-view-tracking-component').length;
+
+    // at the click of a button, we send a notification to ophan and clean up
+    // registered listeners to prevent duplicate calls
+    const onClick = (evt: Event) => {
+        // Flow will complain if we don't type check
+        if (!(evt.target instanceof Element)) {
+            throw new Error('This will never happen');
+        }
+        const button = evt.target.closest('.js-button');
+        const value = button && button.getAttribute('data-value');
+        if (value) {
+            // Flow will complain if we don't type check
+            if (!(evt.currentTarget instanceof Element)) {
+                throw new Error('This will never happen');
+            }
+            const question = evt.currentTarget;
+            const feedback = question.lastElementChild;
+            question.removeEventListener('click', onClick);
+            fastdom
+                .write(() => {
+                    question.classList.add('is-answered');
+                    [...question.querySelectorAll('.js-button')].forEach(b =>
+                        b.setAttribute('hidden', 'hidden')
```

Comment:
If you're ok with sacrificing users of assistive tech on top of IE9/10 over readability, I'll stick to following the html standard. I can address it later if we get complaints?